bin/run-tests-in-docker.sh

```diff
@@ -23,6 +23,7 @@ docker run \
     --read-only \
     --mount type=bind,src="${PWD}/tests",dst=/opt/test-runner/tests \
     --mount type=tmpfs,dst=/tmp \
+    --mount type=tmpfs,dst=/root/.dub \
```

Comment:
This won't work in production as this flag won't be passed. Would not passing the readonly flag fix it? If so, feel free to remove that flag and see if that fixes it, as we're technically not yet passing that flag in production.

Reply:
I was getting an error about that folder being a read only location even if i made the whole image writable.